doc/specs/vulkan/chapters/vertexpostproc.txt

```diff
@@ -680,32 +680,28 @@ The floating-point viewport bounds are represented with an
 
 .Valid Usage
 ****
-  * [[VUID-VkViewport-width-01227]]
-    pname:width must: be greater than `0.0` and less than or equal to
+  * pname:width must: be greater than `0.0`
+  * pname:width must: be less than or equal to
     sname:VkPhysicalDeviceLimits::pname:maxViewportDimensions[0]
-ifndef::VK_KHR_maintenance1[]
-  * [[VUID-VkViewport-height-01228]]
-    pname:height must: be greater than `0.0` and less than or equal to
+ifndef::VK_KHR_maintenance1,VK_AMD_negative_viewport_height[]
+  * pname:height must: be greater than `0.0`
+endif::VK_KHR_maintenance1,VK_AMD_negative_viewport_height[]
+ifdef::VK_KHR_maintenance1,VK_AMD_negative_viewport_height[]
+  * pname:height must: be greater than or equal to
+    -sname:VkPhysicalDeviceLimits::pname:maxViewportDimensions[1]
```

Comment:
Hmm, it's kind of unclear to put the `-` in front of this given that it doesn't even remotely look like a mathematical statement otherwise - I think it'd make more sense to say explicitly "the negation of ..maxViewportDimensions[1]" or similar language.

Granted that's sort of an existing problem, but would be good to fix whilst you're here - the output just looks broken.

Reply:
That's probably not a bad idea. Would just wording out "minus" be better?

Reply:
"equal to minus blah" is weird phrasing IMO, I think that only works with numerical constants (e.g. "minus five").

"negative" sounds ok in speech but I think might be grammatically incorrect in this particular case...? Not sure. "Negative value" maybe?

I suggested "the negation of" because it's accurate, and mirrors where we've put "the sum/product/difference/quotient of" elsewhere in valid usage. I think in VU it pays to be formal, though I'd use something less formal if I were to express it elsewhere.

Reply:
Hmm. Alternatively "an absolute value of height must be less than or equal to maxViewportDimensions[1]".

Reply:
That would work, and wouldn't even need to be conditional based on extension.

Reply:
Alright, updated that way.